test/iron-icons.html

```diff
@@ -58,13 +59,15 @@
         expect(iconNames).to.not.be.empty;
 
         for (var j = 0; j < iconNames.length; j++) {
-          var parts = iconNames[j].split(':');
+          var iconName = iconNames[j];
+          var parts = iconName.split(':');
+
           expect(parts).to.have.length(2);
           expect(parts[0]).to.equal(iconsetNames[i]);
           expect(parts[1]).to.have.length.at.least(1);
-          expect(allIcons.hasOwnProperty(parts[1])).to.be.false;
```

Comment:
I noticed that this test failed while running locally so I fixed it to look at the entire name instead of just a part. However, after just now doing some more digging, it looks like this test is currently broken, yet unreported, because there are two icons with the same name following the `:` ([here](https://github.com/PolymerElements/iron-icons/blob/8d999f711c42366c7f81247eddbd11882b473533/places-icons.html#L30) and [here](https://github.com/PolymerElements/iron-icons/blob/8d999f711c42366c7f81247eddbd11882b473533/notification-icons.html#L53)) but it looks like [this test was added](https://github.com/PolymerElements/iron-icons/pull/60) after [the tests were disabled](https://github.com/PolymerElements/iron-icons/pull/55) without reenabling them. I'll reenable the tests.